src/components/Dashboard/DeckLibrary.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState, useEffect} from "react";
+import axiosWithAuth from "../../helpers/axiosWithAuth";
+import styles from "./DeckLibrary.module.css";
+import {Link} from "react-router-dom";
+import {Card, Icon} from "antd";
+
+export default function DeckLibrary() {
+  const [decks, setDecks] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    axiosWithAuth()
+      .get("http://localhost:4003/api/decks")
+      .then(res => {
+        console.log(res);
+        setDecks(res.data.data);
+        return res;
+      })
+      .then(res => {
+        setLoading(false);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }, []);
```

Comment:
We agreed on using async await syntax. Also, it should probably be connected to redux and side-effects should be managed by redux actions.